### ISSUE TITLE
Bulleted lists are not navigable with screen reader shortcut keys.

### ISSUE THREAD

**Issue by buddy-harlow** [Bixal/bixal-site-drupal#246]
## Location: https://www.bixal.com/ in the 'Our Clients and Partners' section

- As a screen reader user, when I press the ‘L’ shortcut key, these bulleted lists are skipped over.

![Image](https://github.com/user-attachments/assets/1fd65f0e-5965-4cdb-b1f2-966a9e804a46)

## Definition of done
As a screen reader user, when I press the 'L' shortcut key, I should be able to navigate directly to all lists on the page.

Also noticed that the bullets under the 'Health' section are a different color and not black like the bullets in the other sections.

**Comment by buddy-harlow** [comment]
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl#accessibility

Figure out what screenreaders were used for testing, refer to link above ^ 

If it's a major accessibility issue we can remediate/determine a solution, potentially an unordered list.

**Comment by mejiaj** [comment]
Used the accessibility information listed in [description list support](https://adrianroselli.com/2022/12/brief-note-on-description-list-support.html) that MDN links to.

| Browser        | Screen reader | List navigation |
| -------------- | ------------- | --------------- |
| Firefox        | NVDA          | ✅              |
| Chrome         | JAWS          | ✅              |
| Edge           | Narrator      | unknown         |
| Safari         | VoiceOver     | ✅[^1]          |
| Chrome android | Talkback      | ✅              |
| Safari iOS     | VoiceOver     | ❌ |


[^1]: Works with <kbd>VO</kbd> + <kbd>X</kbd>

## Conclusion

Overall support for navigating definition list is pretty good. I recommend we either:
1. **Leave as it is.** Support is only going to increase and it's a semantic application of the element.
2. **Use a list element.** Requires restructuring markup and styles. An hour task with testing included.
